document isotope labelling as of discussion #95 (#96)
Rappsilber-Laboratory/XiSearch: README.md

     loss:CleavableCrosslinkerPeptide:MASS:0;Name:0
     loss:CleavableCrosslinkerPeptide:MASS:82.04186;NAME:S

#### Isotope labelling

> Isotope labelling is supported in xiSEARCH as a custom setting. For samples that are fully labelled, or containing some heavy labelled protein, the LABEL can be defined by updating monoisotopic mass of each amino acid. For example, for 15N:

Reviewer: ```suggestion
 xiSEARCH has some preliminary support for isotope labelling as a custom setting via LABEL settings. For the search each protein is present twice - ones unlabelled and ones fully labelled. (so only heavy - light schema is supported).  A label needs to be defined for every labelled Amino Acid. The LABEL can be defines the monoisotopic mass of the labelled amino acid (i.e. no delta mass). 
An example for 15N labelling defintion would look like this:
```

> Isotope labelling is supported in xiSEARCH as a custom setting. For samples that are fully labelled, or containing some heavy labelled protein, the LABEL can be defined by updating monoisotopic mass of each amino acid. For example, for 15N:

Reviewer: ```suggestion
Isotope labelling is supported in xiSEARCH as a custom setting. For samples that are fully labelled, or containing some heavy labelled protein, the LABEL can be defined with a new monoisotopic mass of each affected amino acid. For example, for 15N on every amino-acid:
```

    LABEL:HEAVY::SYMBOL:Dn15;MODIFIED:D;MASS:116.023978035
    LABEL:HEAVY::SYMBOL:En15;MODIFIED:E;MASS:130.039628105
    LABEL:HEAVY::SYMBOL:Fn15;MODIFIED:F;MASS:148.065448915
    LABEL:HEAVY::SYMBOL:Gn15;MODIFIED:G;MASS:58.018498705
    LABEL:HEAVY::SYMBOL:Hn15;MODIFIED:H;MASS:140.050016785
    LABEL:HEAVY::SYMBOL:In15;MODIFIED:I;MASS:114.081098985
    LABEL:HEAVY::SYMBOL:Kn15;MODIFIED:K;MASS:130.08903299
    LABEL:HEAVY::SYMBOL:Ln15;MODIFIED:L;MASS:114.081098985
    LABEL:HEAVY::SYMBOL:Mn15;MODIFIED:M;MASS:132.037519615
    LABEL:HEAVY::SYMBOL:Nn15;MODIFIED:N;MASS:116.03699741
    LABEL:HEAVY::SYMBOL:Pn15;MODIFIED:P;MASS:98.049798845
    LABEL:HEAVY::SYMBOL:Qn15;MODIFIED:Q;MASS:130.05264748
    LABEL:HEAVY::SYMBOL:Rn15;MODIFIED:R;MASS:160.08925093
    LABEL:HEAVY::SYMBOL:Sn15;MODIFIED:S;MASS:88.029063405
    LABEL:HEAVY::SYMBOL:Tn15;MODIFIED:T;MASS:102.044713475
    LABEL:HEAVY::SYMBOL:Vn15;MODIFIED:V;MASS:100.065448915
    LABEL:HEAVY::SYMBOL:Wn15;MODIFIED:W;MASS:188.07338292
    LABEL:HEAVY::SYMBOL:Yn15;MODIFIED:Y;MASS:164.060363545
    LABEL:HEAVY::SYMBOL:Un15;MODIFIED:U;MASS:151.950668375
    LABEL:HEAVY::SYMBOL:On15;MODIFIED:O;MASS:240.138831835

> This will then be used in the search to match heavy version of proteins, while still allowing the program to match the light versions. This can also be used to support SILAC labelling. In this case, only the amino acids affected by the label need to be redefined.

Reviewer: ```suggestion
This will then be used in the search to match heavy version of proteins, while still allowing the program to match the light versions. This can also be used to support SILAC labelling. In this case, only the amino acids affected by the label need to be redefined. Customary would be to name the label  something like K8 if the labelled amino-acid is 8 Dalton heavier then the unlabelled.
```
